libbeat/docs/outputconfig.asciidoc

```diff
@@ -633,10 +633,10 @@ Note: If set to 0, no ACKs are returned by Kafka. Messages might be lost silentl
 
 The number of seconds to wait for new events between two producer API calls.
 
-===== tls
+===== ssl
 
-Configuration options for TLS parameters like the root CA for Kibana connections. See
-<<configuration-output-tls>> for more information.
+Configuration options for SSL parameters like the root CA for Kibana connections. See
```

Comment:
Perhaps I miss something here, but shouldn't Kibana be replaced by elasticsearch?